synfig-studio/src/synfigapp/actions/valuedescexport.cpp

```diff
@@ -58,7 +58,7 @@ using namespace Action;
 
 ACTION_INIT_NO_GET_LOCAL_NAME(Action::ValueDescExport);
 ACTION_SET_NAME(Action::ValueDescExport,"ValueDescExport");
-ACTION_SET_LOCAL_NAME(Action::ValueDescExport,N_("Export Value"));
+ACTION_SET_LOCAL_NAME(Action::ValueDescExport,N_("Export Value to project library"));
```

Comment:
I really don't know if it helps :/

Reply:
I believe this references Library Panel where all exported values stored. That way user have a hint where to look for list of exported values. ^__^